checker/tests/lock/TestTreeKinds.java

```diff
@@ -74,248 +135,343 @@ Object getFoo() {
   @GuardedBy("lock")
   myEnumType myEnum;
 
+/* TODO Reenable this after @GuardedByName annotation is implemented that can guard primitives.
```

Comment:
Should you move these tests to a separate file, that get's skipped? Then you don't have to list so many TODO comments for each skipped test. Also, file an issue so you don't forget.